tests/EventListener/RespondListenerTest.php

```diff
@@ -78,6 +78,7 @@ public function testCreate200Response()
         $this->assertEquals('Accept', $response->headers->get('Vary'));
         $this->assertEquals('nosniff', $response->headers->get('X-Content-Type-Options'));
         $this->assertEquals('deny', $response->headers->get('X-Frame-Options'));
+        $this->assertNull($response->headers->get('Location'));
```

Comment:
That doesn't test the current change as this a `GET` request. Should I create a `testCreate200PostResponse` method or something like that?